### PR TITLE
PD-247 automate metrics

### PR DESCRIPTION
**Pull request by nathan-px**

### CODE REVIEW

**Comment by nathan-px** [comment]
@sangleganesh @jrivera-px Let's use this PR to figure out where we should place the metrics reference material.

**Comment by jrivera-px** [comment]
Are we generating this file with same checkin name?

File: content/install-with-other/operate-and-maintain/monitoring/prometheus/index.md [hunk 1]
```diff
@@ -36,71 +36,6 @@ A curl request on port 9001 also shows the stats:
 
 ![Curl Request on 9001](/img/curl-request-on-9001.png "Curl Request on 9001")
 
-## Storage and Network stats
```
Comment (nathan-px) [inline content/install-with-other/operate-and-maintain/monitoring/prometheus/index.md]: @sangleganesh I removed these as part of the new metrics page. Can you just confirm the new metrics replace these?

**Comment by sangleganesh** [comment]
@nathan-px change looks good.

**Pull request merged**